github-for-jira.sd.yml

```diff
@@ -329,6 +330,11 @@ environmentOverrides:
           ComparisonOperator: GreaterThanOrEqualToThreshold
           Statistic: Sum
     resources:
+      - type: redisx
+        name: cache
+        attributes:
+          size: 5000
```

Comment:
We should add an override for staging as well. It seems Micros forces size:200 for all non-prod instances unless you have an override (I'll PM you the link)

Reply:
I've set it to 500 by default, which I really doubt we'll exceed that in staging for the very few people with access to it.